modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectDetails/AccountRestrictionContainer.tsx

```diff
@@ -16,8 +16,8 @@ import {defaultLanguageId} from '../../utils/constants';
 interface AccountRestrictionContainerProps {
 	errors: FormError<ObjectDefinition>;
 	isApproved: boolean;
+	isLinkedObjectDefinition?: boolean;
```

Comment:
here I change to `isLinkedObjectDefinition` in order to follow the pattern of `isApproved` and `isRootDescendantNode`